extensions/keepassxc/src/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { ActionPanel, closeMainWindow, List, showToast, ToastStyle } from "@raycast/api";
+import { loadEntries, copyAndPastePassword, copyPassword, copyUsername } from "./utils/keepassLoader";
+import { useState, useEffect } from "react";
+
+const errorHandler = (e: { message: string }) => {
+  let invalidPreference = "";
+  if (e.message.includes("Invalid credentials were provided")) {
+    invalidPreference = "Password";
+  } else if (e.message.includes("keepassxc-cli: No such file or directory")) {
+    invalidPreference = "Path of KeepassXC.app";
+  } else if (
+    e.message.includes("Failed to open database file") ||
+    e.message.includes("Error while reading the database: Not a KeePass database")
+  ) {
+    invalidPreference = "Keepass Database File";
+  }
+  let toastTitle = "Error";
+  let toastMessage = e.message.trim();
+  if (invalidPreference !== "") {
+    toastTitle = `Invalid Preference: ${invalidPreference}`;
+    toastMessage = "Please Check Extension Preference.";
+  }
+  showToast(ToastStyle.Failure, toastTitle, toastMessage);
+};
+
+export default function Command() {
+  const [entries, setEntries] = useState<string[]>();
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    loadEntries()
+      .then(setEntries)
+      .catch(errorHandler)
+      .then(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <List isLoading={isLoading} searchBarPlaceholder="Type to Search in KeepassXC" throttle={true}>
+      {entries?.map((entry, i) => (
+        <List.Item
+          key={i}
+          title={entry.split("/")[entry.split("/").length - 1]}
+          accessoryTitle={
+            entry.split("/").length > 2
+              ? entry
+                  .split("/")
+                  .slice(1, entry.split("/").length - 1)
+                  .join("\t")
+              : ""
+          }
+          keywords={entry.split("/").slice(1)}
+          actions={
+            <ActionPanel>
+              <ActionPanel.Item
+                title="Copy and Paste Password"
```

Comment:
```suggestion
                title="Paste"
                icon={Icon.TextDocument}
```
Would make more sense to have it only paste with this action (rewrite logic of copyAndPastePassword)